.github/workflows/build.yaml

```diff
@@ -5,8 +5,6 @@ on:
     branches:
       - main
   pull_request:
-    branches:
-      - main
```

Comment:
so this will enable CI run on any PR?

Reply:
Yep. CI will run for any pull request, and for main whenever it is pushed.